central/telemetry/phonehome/interceptor.go

```diff
@@ -17,6 +18,8 @@ import (
 	"google.golang.org/grpc"
 )
 
+const grpcGatewayUserAgentHeader = runtime.MetadataPrefix + "user-agent"
```

Comment:
```suggestion
const grpcGatewayUserAgentHeader = runtime.MetadataPrefix + "User-Agent"
```
I understand that `metadata.MD` converts keys to lowercase but this is not obvious here, is not common knowledge, and who knows when it changes. [Canonical form](https://github.com/grpc-ecosystem/grpc-gateway/blob/8952e38d5addd28308e29c272c696a578aa8ace8/runtime/context.go#L288) is "User-Agent".